### ISSUE TITLE
[BUG] No italics in subtitles in HBO Spain

### ISSUE THREAD

**Issue by Paco8** [arvvoid/plugin.video.hbogoeu#27]
I'm using HBO Spain. I noticed that subtitles never use the italic format (normally used for songs or words in other languages).
In the original subtitle files the &lt;span&gt; tag is used to mark them but it seems the conversion done in ttml2srt.py ignores that tag.

My knowledge about python is almost zero but I managed to write this patch which seems to fix the problem:

```
--- ttml2srt-orig.py    2019-09-03 22:19:58.034970566 +0200
+++ ttml2srt.py    2019-09-03 22:20:23.030656327 +0200
@@ -25,6 +25,9 @@
     def extract_dialogue(self, nodes):
         dialogue = ''
         for node in nodes:
+            if node.localName == 'span' and node.hasChildNodes():
+                dialogue = dialogue + "<i>" + extract_dialogue(node.childNodes) + "</i>"
+                continue
             if node.localName == 'br':
                 dialogue = dialogue + '\n'
             elif node.nodeValue:

```

**Comment by arvvoid** [comment]
Tnx for reporting, will look into it

**Comment by Paco8** [comment]
I made a mistake with the patch. I think this is the correct one:

```
--- ttml2srt-orig.py	2019-09-05 18:02:25.308705525 +0200
+++ ttml2srt.py	2019-09-05 18:00:35.677279630 +0200
@@ -25,6 +25,9 @@
     def extract_dialogue(self, nodes):
         dialogue = ''
         for node in nodes:
+            if node.localName == 'span' and node.hasChildNodes():
+                dialogue = dialogue + "<i>" + self.extract_dialogue(node.childNodes) + "</i>"
+                continue
             if node.localName == 'br':
                 dialogue = dialogue + '\n'
             elif node.nodeValue:

```

**Comment by arvvoid** [comment]
Can you please send a pull request?

**Comment by arvvoid** [comment]
Fixed with pull request https://github.com/arvvoid/plugin.video.hbogoeu/pull/29

Will be included in next relese

**Comment by arvvoid** [comment]
Relesed fix in 2.0.14~beta30
